_taxonomy.rst

```diff
@@ -0,0 +1,69 @@
+.. _taxonomy:
+
+Documentation Taxonomy
+======================
+
+Data Management uses and produces multiple classes of documentation.
+Each class fulfills a specific role, with a consistent management and technical process.
+
+.. _taxonomy-outline:
+
+Document classes
+----------------
+
+The classes of DM documentation are:
+
+Requirements and interface control documentation (LPM, LSE)
+   These documents specify functionality that the Data Management System must deliver to the Project.
+   Similarly, **interface control documents** are agreements between subsystems on functionality that cross subsystem boundaries.
+   The production of this documentation is described in `LPM-19: Change Control Process`_ and its management is described in `LPM-51: Document Management Plan`_.
+
+Change-controlled DM design documentation (LDM)
+   These documents define the **scope** and **budget** of Data Management work that meet the **requirements** and **interface control documentation**.
+   DM work is expected to be consistent with these **design documents**.
+   LDM document authors are responsible for updating design documentation whenever necessitated by revised project requirements, or by evolving implementation choices.
+   Design documents must be approved by the Change Control Board (`LPM-19: Change Control Process`_) before becoming the new baseline.
+   These documents are also governed by `LPM-51: Document Management Plan`_.
+   See :ref:`ldm` for how change-controlled documents are handled by the Data Management Documentation Architecture.
+
+Technical notes (DMTN, SQR)
+   Technical notes are a standardized, durable, document series allowing useful information to be published on demand by any Data Management team member wishing to capture and share knowledge.
```

Comment:
> We may need another group for things like test plans and verification documentation. I'm not sure that DMTNs would be adequate for this.

Can **test plans** be written as LDM documents?

Code that carries out test measurements will be documented with User Guides, and the LDM document and user guides can link between each other.

**Verification documentation** is an interesting subject. In [Figure 1](https://ldm-493.lsst.io/v/v1/index.html#fig-information-flow) I envisioned a class of 'validation documentation' but I didn't include this in the taxonomy or in a dedicated section because I'm not sure what the requirements for this documentation will be. @frossie has talked about dynamically-generating validation/verification documentation as runnable Jupyter notebooks. We can certainly handle this with the existing infrastructure. Systems engineering might have additional requirements and we should engage with them.

***

Regarding lower-level, non-change-controlled design documentation: does https://ldm-493.lsst.io/v/v1/index.html#relation-between-users-guides-and-design-documentation answer your question? tl;dr is that low-level design docs can be written as technotes, but that content should be deprecated and migrated into the software or system's user guide as it is implemented. There was a discussion about this on Community in relation to documenting FITS headers. The consensus seemed to be that since DM is an open source project, the code's documentation should be comprehensive enough to support users as both 'end users' and 'developers.'